airbyte-integrations/connectors/source-chargify/manifest.yaml

```
@@ -184,24 +356,46 @@ definitions:
             type: PageIncrement
             page_size: 200
             start_from_page: 1
+      incremental_sync:
+        type: DatetimeBasedCursor
+        cursor_field: updated_at
+        cursor_datetime_formats:
+          - "%Y-%m-%d %H:%M:%S"
+          - "%Y-%m-%d %H:%M:%S.%f+00:00"
+          - "%Y-%m-%dT%H:%M:%S.%f%z"
+          - "%Y-%m-%dT%H:%M:%S%z"
+        datetime_format: "%Y-%m-%d %H:%M:%S"
+        start_datetime:
+          type: MinMaxDatetime
+          datetime: "{{ config[\"start_date\"] }}"
```

```suggestion
          datetime: "{{ config.get("start_date") or "2020-01-01" }}"
```
Add a default date to not introduce a breaking change to existing connections.